docs/pages/desktop-access/getting-started.mdx

```diff
@@ -176,6 +192,24 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Publi
   ![Import Teleport CA](../../img/desktop-access/ca.png)
 </Figure>
 
+### Publish the Teleport CA to the Active Directory domain
+
+<Admonition type="note" title="Differences when using AWS Managed Active Directory">
+  When using AWS Managed Active Directory, you should run this command using an account which is part of the
+  `AWS Delegated Domain Administrators` group, such as the AWS-provided `admin` account.
+</Admonition>
+
+On a machine which is joined to your domain and logged in as an account in the `Domain Administrators` group,
+run the two commands below at a PowerShell prompt to publish the Teleport CA to your Active Directory
+domain (using the path to the exported Teleport `user-ca.cer` file that you copied above):
+
+```powershell
+certutil –addstore –f root C:\Path\To\user-ca.cer
```

Comment:
I'm not convinced this first command is needed.

`-addstore` publishes to the local machines local store
`-dspublish` publishes to active directory

Is is possible we don't need addstore?

Reply:
Yes, it's possible - it's just what the AWS team recommended so I ran both.

Reply:
Going to remove this in an edit.